templates/parachain/README.md

```diff
@@ -47,90 +70,143 @@ git clone https://github.com/paritytech/polkadot-sdk-parachain-template.git para
 cd parachain-template
 ```
 
-### Build
+## Starting a Development Chain
+
+### Omni Node
 
-🔨 Use the following command to build the node without launching it:
+[Omni Node](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html) can
+be used to run the parachain template's runtime. `polkadot-omni-node` binary crate usage is described at a high-level
+[on crates.io](https://crates.io/crates/polkadot-omni-node).
+
+#### Install `polkadot-omni-node`
+
+Please see the installation section at [`crates.io/omni-node`](https://crates.io/crates/polkadot-omni-node).
+
+#### Build `parachain-template-runtime`
 
 ```sh
-cargo build --release
+cargo build -p parachain-template-runtime --release
 ```
 
-🐳 Alternatively, build the docker image:
+#### Install `staging-chain-spec-builder`
+
+Please see the installation section at [`crates.io/staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder).
+
+#### Use chain-spec-builder to generate the chain_spec.json file
 
 ```sh
-docker build . -t polkadot-sdk-parachain-template
+chain-spec-builder create --relay-chain "rococo-local" --para-id 1000 --runtime \
```

Comment:
Maybe would be good to mention that `"rococo-local"` is our relay chain name given in: 
https://github.com/paritytech/polkadot-sdk/blob/ef957f2787b30b7d264ceceb7bc1bde3bcf22753/templates/parachain/zombienet-omni-node.toml#L3 ?

btw: I am actually wondering if these two names need to be synced? Is this value used somewhere in communication with relay-chain? (cc @skunert / @kianenigma )

Reply:
It should be sync-ed:

- `relay_chain` in the parachain chain spec is used AFAIK for the internal relay chain node that the parachain node runs. The parachain node will run `polkadot --chain xxx`
- `chain = "rococo-local"` in zombienet.toml is the actual node that you run, `polkadot --chain yyy`.